packages/create-cloudflare/templates/hello-world/py/pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[project]
+name = "<TBD>"
```

Comment:
This `<TBD>` doesn't get updated by `create-cloudflare` automatically, unlike those in `wrangler.jsonc` for instance. We should fix that

Reply:
They do now!